Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -202,7 +202,7 @@ void doChargeDepositionShapeN (const GetParticlePosition& GetPosition,
         }
         );
 #if defined(WARPX_USE_GPUCLOCK)
-        if( load_balance_costs_update_algo == LoadBalanceCostsUpdateAlgo::GpuClock) {
+        if(cost && load_balance_costs_update_algo == LoadBalanceCostsUpdateAlgo::GpuClock) {
```

Comment:
Awesome, thanks for testing!

We need to find out why `load_balance_costs_update_algo == LoadBalanceCostsUpdateAlgo::GpuClock` results in `true`even though we select `Timers`. But thats orthogonal to fix.